src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/ValidationResultExceptionConverter.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class ValidationResultExceptionConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            return typeof(Exception).IsAssignableFrom(objectType);
```

Comment:
Do we have to use the converter for all exceptions, when only certain HttpClient exceptions are failing to deserialize?

Reply:
there's a default one (by JSON.NET) that this calls

```
return serializer.Deserialize(reader, objectType);
```